src/renderer/shared/api/governance/off-chain/service/deligationService.ts

```diff
@@ -0,0 +1,64 @@
+import { GraphQLClient } from 'graphql-request';
+
+import { dictionary } from '@/shared/lib/utils';
+import { type Chain } from '@shared/core';
+import { type DelegateAccount, type DelegateDetails, type DelegateStat, type DelegationApi } from '../lib/types';
+
+import { GET_DELEGATE_LIST } from './delegation/queries';
+
+const DELEGATE_REGISTRY_URL =
+  'https://raw.githubusercontent.com/novasamatech/opengov-delegate-registry/master/registry';
+
+async function getDelegatesFromRegistry(chain: Chain): Promise<DelegateDetails[]> {
+  try {
+    const normalizedName = chain.name.toLowerCase();
+
+    const response = await fetch(`${DELEGATE_REGISTRY_URL}/${normalizedName}.json`);
+
+    return (await response.json()) as DelegateDetails[];
+  } catch (e) {
+    return [];
+  }
```

Comment:
I think it can be mush simpler with old good promise api
```suggestion
const normalizedName = chain.name.toLowerCase();

return fetch(`${DELEGATE_REGISTRY_URL}/${normalizedName}.json`)
  .then((res) => res.json())
  .catch(() => [])
```